client/packages/inventory/src/Stocktake/ListView/CreateStocktakeButton.tsx

```diff
@@ -100,7 +100,11 @@ export const CreateStocktakeButton: React.FC<{
       id: FnUtils.generateUUID(),
       description,
       masterListId: masterListId ? masterListId : undefined,
-      locationId: locationId ? locationId : undefined,
+      location: locationId
+        ? {
+            value: locationId,
+          }
+        : undefined,
```

Comment:
This change is an exception to calling structuring the graphql call in the api with

```location: setOptionString('id' as keyof typeof line.location, {
        ...line.location,
      }),
      ```
      In this case the location is passed with just an ID rather than the LocationNode structure, so it didn't fit the same pattern